README.md

```diff
@@ -81,6 +81,22 @@ Run individual tests in the foreground and report.
 rainforest run <test_id1> <test_id2>
 ```
 
+Run a run group.
+
+⚠️ This uses the configuration defined in the run group (environment, browsers, crowd, location). If you wish to run tests from a run group without using the run group's configuration, you will need to use the Rainforest API directly, passing a `run_group_id` parameter to [the `POST /runs` endpoint](https://app.rainforestqa.com/docs#!/runs/post1Runs). ⚠️
```

Comment:
`This uses the configuration defined in the run group: **crowd**`, let's keep the comment, but actually we have bug/feature here.
Even if a run group in web has Automation crowd, when it's run `rainforest-cli run --run-group id` it will use Testers actually.
To run with automation `--crowd` parameter is required

Reply:
Yep we've run into this bug... but that's on the backend, not the CLI 👍

Reply:
Looking into this some more... the bug _is_ on the CLI: https://github.com/rainforestapp/rainforest-cli/blob/05da067539ddce74f2f00729f7ccf7146b77795d/rainforest-cli.go#L188